test.cpp

```diff
@@ -934,7 +934,7 @@ static void define_va_opt_3()
                          "err()";
 
     ASSERT_EQUALS("", preprocess(code2, &outputList));
-    ASSERT_EQUALS("file0,1,syntax_error,failed to expand 'err', Missing parenthesis for __VA_OPT__(content)\n",
+    ASSERT_EQUALS("file0,1,syntax_error,Failed to parse #define, Missing opening parenthesis for __VA_OPT__\n",
```

Comment:
hmm.. I personally have a better feeling for "expand" than "parse" is this reported for unexpanded macros also? 

Imho it seems helpful to say the macro name.

I like that you say "opening" / "closing" :+1:

Reply:
The error happens on the `#define` directive during parsing and not during macro expansion, so the error is emitted even if the macro is never expanded. This is the same behavior as GCC and Clang. I've update the message to include the macro name.